docs/Guides/Ecosystem.md

```diff
@@ -682,7 +682,7 @@ middlewares into Fastify plugins
   [uws](https://github.com/uNetworking/uWebSockets).
 - [`fastify-xml-body-parser`](https://github.com/NaturalIntelligence/fastify-xml-body-parser)
   Parse XML payload / request body into JS / JSON object.
-- [`fastify-xray`](https://github.com/jeromemacias/fastify-xray) Fastify plugin
+- [`fastify-xray`](https://github.com/aws/aws-xray-sdk-node/tree/master/sdk_contrib/fastify) Fastify plugin
```

Comment:
Your document is linked to https://www.npmjs.com/package/aws-xray-sdk-fastify
Please use the proper package name.

Since the package is belong to `jeromemacias`, I would like to ask if you wanted to be removed in the list and replaced by the `aws` version.
cc @jeromemacias

Reply:
Package name updated.

Reply:
Please arrange the plugin name to follow the alphabetical order.
Thanks.

Reply:
Done!